app/electron/main.ts

```diff
@@ -402,6 +507,17 @@ function startElecron() {
       },
     });
 
+    setMenu(mainWindow, currentMenu);
+
+    // Load the full menu on timeout.
+    setTimeout(() => {
+      if (!loadFullMenu) {
+        console.info('Timed out waiting for plugins to be loaded, so loading full menu now');
+        loadFullMenu = true;
+        setMenu(mainWindow, currentMenu);
+      }
+    }, defaultFullMenuLoadTimeout);
```

Comment:
It will work fine because the plugin has received and processed the menu spec that we're showing. So if the plugin e.g. replaces the website URL, it'll show it fine.